app/uk/gov/hmrc/agentmappingfrontend/views/not_enrolled_template.scala.html

```diff
@@ -0,0 +1,29 @@
+@*
+ * Copyright 2017 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.agentmappingfrontend.config.AppConfig
+@import uk.gov.hmrc.agentmappingfrontend.views.html
+
+
+@()(implicit request: Request[_], messages: Messages, appConfig: AppConfig))
+
+@html.main_template(appConfig, title = "Not Enrolled", bodyClasses = None) {
+    <h1 class="heading-xlarge">
+        Not Enrolled
+    </h1>
+
+    You are not enrolled.
```

Comment:
is this a true statement if the user has a pending enrolment?

I think the story might be missing an acceptance criteria - it has one for enrolled and activated, and one for not enrolled, but it doesn't have one for "enrolled but not activated"